packages/app/src/server/routes/apiv3/customize-setting.js

```diff
@@ -13,6 +15,13 @@ const { body, query } = require('express-validator');
 
 const ErrorV3 = require('../../models/vo/error-apiv3');
 
+const apiLimiter = rateLimit({
+  windowMs: 15 * 60 * 1000, // 15 minutes
+  max: 10, // limit each IP to 10 requests per windowMs
+  message:
+    'Too many requests were sent from this IP. Please try a password reset request again on the password reset request form',
+});
+
```

Comment:
I added this lines because the code scanning bot gives the warning `Missing rate limiting`, Can we just  ignore it? 
![image](https://user-images.githubusercontent.com/92426728/175896543-c2b8c2e8-84cb-41b4-b913-040cb6d4af44.png)